doc/classes/Node.xml

```diff
@@ -257,6 +257,12 @@
 				[b]Note:[/b] If you only want to find the first descendant node that matches a pattern, see [method find_child].
 			</description>
 		</method>
+		<method name="find_exposed_children" qualifiers="const">
+			<return type="Node[]" />
+			<param index="0" name="recursive" type="bool" default="true" />
+			<description>
```

Comment:
This needs to be documented, as does the one below

Reply:
Hi, do you mean like creating a pull request for the godot-docs?  or some other area to be documented

Reply:
Yep the description 🙂